core/DSGToolsProcessingAlgs/algRunner.py

```diff
@@ -161,15 +161,32 @@ def runDsgToolsClean(self, inputLyr, context, feedback = None, onlySelected = Fa
             'MINAREA': minArea,
             'FLAGS' : flags
         }
-        output = processing.run('dsgtools:cleangeometries', parameters, context = context, feedback = feedback)
+        output = processing.run('dsgtools:cleangeometries', parameters, context=context, feedback=feedback)
         return output['OUTPUT']
-    
-    def runDouglasSimplification(self, inputLyr, threshold, context, feedback = None, snap=None, minArea=None, iterations=None, type=None, returnError=False):
+
+    def runDouglasSimplification(self, inputLyr, threshold, context,
+                                 feedback=None, snap=None, minArea=None,
+                                 iterations=None, type=None, returnError=False,
+                                 flags=None):
+        """
+        Runs simplify GRASS algorithm
+        :param inputLyr: (QgsVectorLayer) layer to be dissolved.
+        :param method: (QgsProcessingParameterEnum) which algorithm would be
+            used to simplify.
+        :param threshold: (QgsProcessingParameterNumber) scale tolerance(m).
```

Comment:
Nas docstrings pode ser um pouco mais verboso, não tem problema: ao invés de '(m)', 'in meters'. MAS, certeza que não seria em unidades de mapa?